### PR TITLE
Copy .editorconfig file and relevant sections in CONTRIBUTING.md from otel-java repo

### PR DESCRIPTION
**Pull request by iNikem**
See https://github.com/open-telemetry/opentelemetry-java/issues/1126

### CODE REVIEW

**Comment by trask** [comment]
Commenting to re-trigger CLA check

**Comment by iNikem** [comment]
@trask @tylerbenson @jkwatson @safris can you merge this PR, please?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -40,17 +40,40 @@ Then Gradle task rule will kick in and do the following:
...
+
+or on Windows
+
+`gradlew.bat googleJavaFormat`
+
+#### Pre-commit hook
+To completely delegate code style formatting to the machine, 
+you can add [git pre-commit hook](https://git-scm.com/docs/githooks).
+We provide an example script in `buildscripts/pre-commit` file.
+Just copy or symlink it into `.git/hooks` folder.
```
Comment (tylerbenson) [inline CONTRIBUTING.md:70]: include suggested command?

Comment (iNikem) [reply CONTRIBUTING.md:70]: Include an example of `cp` command? I expect any developer who is wishing to contribute to this project knows how to do that.

**Comment by iNikem** [comment]
We already use google format, so nothing changes.

File: CONTRIBUTING.md (hunk 1 above)
Comment (tylerbenson) [reply CONTRIBUTING.md:70]: I was thinking the symlink, but good point...

**Pull request merged**